enzyme/test/Integration/ForwardError/err_empty.c

```diff
@@ -0,0 +1,20 @@
+// RUN: %clang -std=c11 -g -O0 %s -S -emit-llvm -o -  %loadClangEnzyme -Xclang -verify
+// RUN: %clang -std=c11 -g -O1 %s -S -emit-llvm -o -  %loadClangEnzyme -Xclang -verify
+// RUN: %clang -std=c11 -g -O2 %s -S -emit-llvm -o -  %loadClangEnzyme -Xclang -verify
+// RUN: %clang -std=c11 -g -O3 %s -S -emit-llvm -o -  %loadClangEnzyme -Xclang -verify
+// RUN: if [ %llvmver -ge 12 ]; then %clang -std=c11 -g -O0 %s -S -emit-llvm -o -  %newLoadClangEnzyme -Xclang -verify; fi
+// RUN: if [ %llvmver -ge 12 ]; then %clang -std=c11 -g -O1 %s -S -emit-llvm -o -  %newLoadClangEnzyme -Xclang -verify; fi
+// RUN: if [ %llvmver -ge 12 ]; then %clang -std=c11 -g -O2 %s -S -emit-llvm -o -  %newLoadClangEnzyme -Xclang -verify; fi
+// RUN: if [ %llvmver -ge 12 ]; then %clang -std=c11 -g -O3 %s -S -emit-llvm -o -  %newLoadClangEnzyme -Xclang -verify; fi
+
+extern double __enzyme_error_estimate(void*, double, double);
+
+double unknown(double in);
+
+double g(double in) {
+    return unknown(in); // expected-error {{Enzyme: No forward mode derivative found for unknown}}
```

Comment:
could we change the error message to "no forward mode error function found" or something?

Reply:
I believe it's hardcoded here: https://github.com/EnzymeAD/Enzyme/blob/main/enzyme/Enzyme/EnzymeLogic.cpp#L4745

Reply:
Ah yes, maybe we could do an if statement there where if mode == ForwardError ... else ...

Reply:
> Ah yes, maybe we could do an if statement there where if mode == ForwardError ... else ...

Ok done